app/models/govuk_publishing_components/component_doc.rb

```diff
@@ -1,25 +1,19 @@
 module GovukPublishingComponents
-  ComponentDoc = Struct.new(:id, :name, :description, :body, :accessibility_criteria, :fixtures) do
-    def self.get(id)
-      component = fetch_component_doc(id)
-      self.build(component)
-    end
-
-    def self.all
-      fetch_component_docs.map { |component| build(component) }
-    end
-
-    def self.build(component)
-      fixtures = component[:fixtures].map { |id, data|
-        GovukPublishingComponents::ComponentFixture.new(id.to_s, data)
-      }
-
-      self.new(component[:id],
-               component[:name],
-               component[:description],
-               component[:body],
-               component[:accessibility_criteria],
-               fixtures)
+  class ComponentDoc
```

Comment:
Why have we moved away from a `Struct`? It looks like we've just wound up with more boilerplate code. To make it immutable? We could use something like [`immutable-struct`](https://github.com/stitchfix/immutable-struct)?